### DISCUSSION TITLE
Creating new operation in store commerce

### DISCUSSION THREAD

**Discussion by anil-24** [microsoft/Dynamics365Commerce.Solutions#208] [Q&A]
Hello everyone,

I've encountered an issue with a new operation I created for my extension, and I'm seeking your assistance to resolve it.

### Steps I've Taken:

1. Created a new operation.
2. Updated the manifest.json file to include this operation.
3. Installed the extension.

Please find the screenshots below:
**Error in store commerce**
![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/46122568/979429d3-54ec-495c-9d61-70861c827cad)

**manifest.json**
![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/46122568/b4403c57-4864-4ece-8ea8-371edd6c9dff)

**Comment by zhangguanghuib** [comment]
Please check the sample here https://github.com/zhangguanghuib/NewCommerceSDK/tree/main/POS_Samples/Return.Restriction/Pos

**Comment by RethaLam** [comment]
Is it just the screenshot or do you have a space after the slash in your manifest file? If it is a space, then you should remove it.
It should be "Operations/TestOperationRequestFactory" and "Operations/TestOperationRequestHandler": no space
